### PR TITLE
chore: add ightk & zalopay

### PR DESCRIPTION
**Pull request by duyunderdog**
In my work experience, these are the 2 applications used to build the system backend

### CODE REVIEW

**Comment by tasszz2k** [comment]
have you worked for both companies GHTK and Zalopay?

File: README.md [hunk 1]
```diff
@@ -63,6 +63,7 @@ If you see anything that is no longer maintained or is not a good fit, please su
 ### I
 
 - [iParking](https://www.iparking.vn/)
+- Ightk ứng dụng tracking đơn hàng GHTK(https://play.google.com/store/apps/details?id=vn.ghtk.ightk.ez)
```
Comment (hmhuan) [inline README.md:66]: I think you should update format here [Ightk ứng dụng tracking đơn hàng GHTK]. Maybe it should be [GHTK]

Comment (tieubao) [reply README.md:66]: agree, please update @duyunderdog

Comment (duyunderdog) [reply README.md:66]: > I think you should update format here [Ightk ứng dụng tracking đơn hàng GHTK]. Maybe it should be [GHTK]

Actually its should be IGHTK not GHTK, but i will change format based on your advice

Comment (duyunderdog) [reply README.md]: @hmhuan check it out 🫶

**Review by hmhuan — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -60,6 +60,9 @@ If you see anything that is no longer maintained or is not a good fit, please su
 
 ### I
 
+- [iParking](https://www.iparking.vn/)
```
Comment (tieubao) [inline README.md:63]: bro please remove this, it's no longer operating

Comment (duyunderdog) [reply README.md]: done @tieubao bro

**Review by tieubao — APPROVED** [review]

**Pull request merged**